NanoProd/crab/Run2_2017/data.yaml

```diff
@@ -11,11 +11,11 @@ SingleElectron_Run2017D: /SingleElectron/Run2017D-UL2017_MiniAODv2-v1/MINIAOD
 SingleElectron_Run2017E: /SingleElectron/Run2017E-UL2017_MiniAODv2-v1/MINIAOD
 SingleElectron_Run2017F: /SingleElectron/Run2017F-UL2017_MiniAODv2-v1/MINIAOD
 
-SingleMuon_Run2017B: /SingleMuon/Run2017B-UL2017_MiniAODv2-v1/MINIAOD
-SingleMuon_Run2017C: /SingleMuon/Run2017C-UL2017_MiniAODv2-v1/MINIAOD
-SingleMuon_Run2017D: /SingleMuon/Run2017D-UL2017_MiniAODv2_GT36-v1/MINIAOD
-SingleMuon_Run2017E: /SingleMuon/Run2017E-UL2017_MiniAODv2-v1/MINIAOD
-SingleMuon_Run2017F: /SingleMuon/Run2017F-UL2017_MiniAODv2_GT36-v1/MINIAOD
+SingleMuon_Run2017B: /SingleMuon/Run2017B-UL2017_MiniAODv2_GT36-v2/MINIAOD
```

Comment:
I am not aware of recommendations for using GT36 for 2017. Could you please add some references? Also, switching for only SingleMuon and not other datasets is inconsistent.

Reply:
I wasn't sure actually, I thought to use these too because they are quite new. I looked around for a bit, and I didn't find anything on CmsTalk (besides other people asking and noone getting an answer). I only found on the [PdmV UL twiki](https://twiki.cern.ch/twiki/bin/view/CMS/PdmVRun2LegacyAnalysis), third bullet, that 2017 is explicitly mentioned. The SingleMuon datasets are the only ones in 2017 that are remade with this tag. They're also found [here](https://pdmv-pages.web.cern.ch/rereco_ul/full_table.html?miniaod_dataset=miniaodv2&nanoaod_dataset=!JME&aod_dataset=15Feb2022) (link from the twiki). Maybe I'll directly ask PdmV about this.

Reply:
ok, let's see what PdmV says. But then there is also the question of consistency if an analysis uses a mix of datasets. As an example in muTau channel one can use OR between single tau, single mu and MET triggers to recover high mass tails, then it will be a mix of GT36 and old events...